src/TaskDeshboard.js

```diff
@@ -0,0 +1,42 @@
+import TaskFilter from "./TaskFilter";
+import TaskList from "./TaskList";
+const TaskDeshboard = () => {
+  const buttons = [
+    { message: "all", isAriaPressed: true },
+    { message: "Active", isAriaPressed: false },
+    { message: "Complete", isAriaPressed: false },
+  ];
+  const cardList = [
+    { chekboxMessage: "Eat", chekboxCheked: true },
+    { chekboxMessage: "Edit", chekboxCheked: false },
+    { chekboxMessage: "Sleep", chekboxCheked: false },
+  ];
+  const numTasks = cardList.length;
+  return (
+    <>
+      <div className="filters btn-group stack-exception">
+        {buttons.map((button) => (
+          <TaskFilter
+            message={button.message}
+            isAriaPressed={button.isAriaPressed}
+          />
+        ))}
+      </div>
+
+      <h2 id="list-heading">{numTasks} tasks remaining</h2>
+      <ul
+        role="list"
+        className="todo-list stack-large stack-exception"
+        aria-labelledby="list-heading"
+      >
+        {cardList.map((card) => (
+          <TaskList
+            chekboxMessage={card.chekboxMessage}
```

Comment:
Muito bom! 😊 
Só um ponto que é uma boa pratica em react, quanto estamos imprimindo listas no react temos que usar uma props `key` no component filho (no caso `TaskItem`), e é usado para otimizar a renderização de listas no react.

Links de referencias:
- https://pt-br.reactjs.org/docs/lists-and-keys.html
- https://pt-br.reactjs.org/docs/lists-and-keys.html#keys